aws_signing_helper/serve.go

```diff
@@ -240,6 +247,9 @@ func AllIssuesHandlers(cred *RefreshableCred, roleName string, opts *Credentials
 				return
 			}
 		} else {
+			if Debug {
+				log.Println("Using previous obtained credentials")
```

Comment:
Minor: should "previous" be "previously" instead?

Reply:
Fixed